src/VirtoCommerce.Platform.Web/Controllers/Api/SecurityController.cs

```diff
@@ -349,6 +349,20 @@ public async Task<ActionResult<SecurityResult>> Create([FromBody] ApplicationUse
                 result = await _userManager.CreateAsync(newUser, newUser.Password);
             }
 
+            if (result.Succeeded && !newUser.Logins.IsNullOrEmpty())
```

Comment:
You need to move this code from here to another place with considering the two cases create and update.

For create
https://github.com/VirtoCommerce/vc-platform/blob/dev/src/VirtoCommerce.Platform.Web/Security/CustomUserManager.cs#L209

For update
https://github.com/VirtoCommerce/vc-platform/blob/dev/src/VirtoCommerce.Platform.Web/Security/CustomUserManager.cs#L177